communications/github-projects.md

```diff
@@ -0,0 +1,55 @@
+## On Github Projects
+
+Since the [Covid Dash development](https://github.com/EQWorks/flaat-dash/projects), we have been using github projects to keep track of progress, issues, and code-specific discussions due to the fast paced nature of the project.
+
+Repos can have multiple projects. Projects can be specific to one extensive task or the whole repo in general - ideally both.
```

Comment:
I think another popular use case is also on one project can be associated with multiple repos.